### ISSUE TITLE
RFE: Include Assignment URL in Todoist Task

### ISSUE THREAD

**Issue by stacksjb** [scottquach/Canvas-Assignments-Transfer-For-Todoist#7]
Rather than just creating a task with the assignment title, include the assignment URL, i.e 

> `[Assignment` Title](https://canvas.instructure.com/courses/10544/assignments/633167)

The value is included in the returned response but can also be generated based on the course and assignment ID.

**Comment by stacksjb** [comment]
I was able to add this pretty easily by changing the task['content'] sections to the following:
`'[' + assignment['name'] + '](' + assignment['html_url'] + ')' + ' Due') `, see my fork.

However, I did notice one quirk - it appears that the html_url is relative to the API being called, so in my case, changing the URL from "canvas.instructure.com" to the appropriate university DOES matter now that that is added - otherwise the URLs don't work. So I'll have to figure out how to set that to some sort of imported variable after all.
